task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -103,5 +101,8 @@ def stop(self):
         self.driver.stop()
         self.driver.join()
 
+    def status(self):
```

Comment:
Do we need this? Currently, all the events are only shared via task_update_queue. We don't store any state in the scheduler, so it will be difficult to implement it.